src/main/java/blackjack/domain/GameResult.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain;
+
+import blackjack.domain.player.Player;
+
+public enum GameResult {
+    WIN("승"),
+    LOSE("패");
+
+    private final String message;
+
+    GameResult(String message) {
+        this.message = message;
+    }
+
+    public static GameResult calculate(Player dealer, Player gamer) {
+        if (isOnlyDealerBursted(dealer, gamer) || hasPlayerHigherScoreThanDealer(dealer, gamer)) {
+            return GameResult.WIN;
+        }
+
+        return GameResult.LOSE;
+    }
+
+    private static boolean isOnlyDealerBursted(Player dealer, Player gamer) {
```

Comment:
찾아봤는데 burst 가 맞나봐요
https://writingexplained.org/burst-vs-bursted
```suggestion
    private static boolean isOnlyDealerBurst(Player dealer, Player gamer) {
```

Reply:
오우... 그렇네요... 감사합니다!!